unic-langid-impl/src/bin/generate_layout.rs

```diff
@@ -28,33 +28,27 @@ fn langid_to_direction_map(path: &str) -> HashMap<LanguageIdentifier, CharacterD
         {
             "right-to-left" => CharacterDirection::RTL,
             "left-to-right" => CharacterDirection::LTR,
+            "top-to-bottom" => CharacterDirection::TTB,
             _ => unimplemented!("Encountered unknown directionality!"),
         };
         result.insert(langid, character_order);
     }
     result
 }
 
-fn check_all_variants_rtl(
+fn check_all_script_variants(
     map: &HashMap<LanguageIdentifier, CharacterDirection>,
-    lang: Option<Language>,
+    exp_dir: &CharacterDirection,
```

Comment:
`nit`: please, make `CharacterDirection` `Copy` don't use a reference here.

Reply:
I tried, but `CharacterDirection` does not implement the `Copy` trait.

Reply:
Add it:
```rust
#[derive(Copy, ...)]
enum CharacterDirection { ... }
```